lib/exercises.rb

```diff
@@ -1,19 +1,57 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: a lot
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
Time complexity 🤣 

It's actually O(n) assuming the words are reasonably short O(n * mlog m) otherwise.